Sources/Markdown/Interpretive Nodes/Aside.swift

```diff
@@ -118,9 +121,35 @@ public struct Aside {
                 todo,
                 version,
                 `throws`,
+                seeAlso,
```

Comment:
Adding `seeAlso` to `allCases` adds test coverage for this new case in `AsideTests.swift`

Reply:
Thanks for the heads-up.